db/migrate/20220901034107_add_user_count_stale_to_channel.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddUserCountStaleToChannel < ActiveRecord::Migration[7.0]
+  def change
+    add_column :chat_channels, :user_count_stale, :boolean, default: false, null: false
```

Comment:
Shouldn't we consider that by default it's stale?

Reply:
Hmmm I don't think so, I think it should only be stale if something changes to increase/decrease this count.